README.md

```diff
@@ -73,6 +114,14 @@ The following repos must be peers of `docs`, should be checked out to an appropr
 - `pulumi-kubernetes`
 - etc.
 
+#### Make errors
+
+`make build` errors are often due to missing dependencies. The following npm packages need to be installed separately on Ubuntu 19.04:
```

Comment:
Running `make ensure` (i.e. `yarn install`) won't install these on Linux?

Reply:
For more background, we haven't been super-consistent, but WAAAAY back when we wanted to have the same setup steps for reach repo:
https://github.com/pulumi/home#building-and-testing

`make`, `make ensure`, and `make install`.

Reply:
@justinvp Unfortunately not. I removed one of the listed dependencies then ran `make ensure`, and got the "package not found" error.

Reply:
What was the specific error that you got? Can you paste the command line output?

Reply:
Ruby is required to run `make ensure`, but `make ensure` takes care of the dependencies, so I've removed that section.